examples/blend_and_shuffle.py

```diff
@@ -46,6 +49,14 @@ def attach_args(
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     ),
 ):
+    parser.add_argument(
```

Comment:
I would not add this argument to the scripts in `examples/`. The examples are meant to be as simple as possible. The only reason why we have the argparse in the first place here is so that the examples can be easily run on SLURM without modifications.